### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by Veskel01** [Siumauricio/rippleui#103] [General]
Hey everyone. I would like to ask what the current status of the project is. Are there any plans for its further support or has the project already been "buried" ? I have just seen RippleUI and am delighted with both its design and ease of implementation

**Comment by Siumauricio** [comment]
Hey @Veskel01 Yes the project will keep receiving features, bug fixing and everything else, I just gave myself a break while bugs or requests arrived, bu I'm back
